doc/classes/Image.xml

```diff
@@ -456,6 +456,15 @@
 			<argument index="1" name="color" type="Color">
 			</argument>
 			<description>
+				Sets the Color of the pixel at (dst.x, dst.y) if the image is locked. Note that the dst values must be integers. Example:
```

Comment:
Should use bbcode tags like other descriptions, something like that:
```
Sets the [Color] of the pixel at [code](dst.x, dst.y)[/code] if the image is locked. Note that the [code]dst[/code] values must be integers. Example:
```

Reply:
Please edit it by amending the commit, see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request).